app/search_builders/report_search_builder.rb

```diff
@@ -8,5 +8,8 @@ class ReportSearchBuilder < Blacklight::SearchBuilder
 
   self.default_processor_chain -= [:add_facetting_to_solr] # remove faceting from reports
 
-  self.default_processor_chain += [:add_date_field_queries] # ensure date field queries work
+  self.default_processor_chain += [
+    :add_access_controls_to_solr_params, # enforce restrictions
+    :add_date_field_queries # ensure date field queries work
```

Comment:
@peetucket unless you're pressing to get this deployed, it might be useful to add a spec as well. if you've already considered and decided not to, happy to hear you out

Reply:
i just added a simple spec that mimics what is there for the existing `SearchBuilder` - it confirms the processor chains have the necessary methods ... there are existing specs that confirm the correctness of those processor chains

building a feature or controller spec is another possibility, but would require confirming that the fixtures have the necessary data to support the checking of varying permissions

Reply:
ok. approving!